### ISSUE TITLE
Changing cloze color

### ISSUE THREAD

**Issue by Asmedeus998** [ssricardo/anki-plugins#71]
![Capture](https://user-images.githubusercontent.com/60872999/85433574-36210e00-b5b7-11ea-86e0-89aa488034a0.PNG)]

![Capture](https://user-images.githubusercontent.com/60872999/85433982-e3942180-b5b7-11ea-8f12-960f5ed86573.PNG)

is there a way to change the color other than white? I try to search it at control text and I don seem to find it.

**Comment by ssricardo** [comment]
Sorry for the late reply.

Do you know CSS?

You can try to use like:


    input.ftb {
      background-color: yellow; /* whatever */
    }

**Comment by Asmedeus998** [comment]
Hi ssricardo,
I know a bit about CSS and I try your suggestion but it still didn't work (putting it in Styling (share between the deck) ). I'll appreciate it if you can make a screenshot and step for making the color change.

**Comment by Asmedeus998** [comment]
Hi ssricardo, 
This might no related to the issue about the color of cloze but I having trouble to cloze a c++ code.
For example 'std::cin >> val' , 
when I try to type in 'std::cin >> val' the color change to green after I only put 'std'.
If I type fully 'std::cin >> val' the color change to red.

The problem fix after I try not to include the 'std::' and the cloze typing go back to normal...
Is there anyway to fix this problem?

**Comment by ssricardo** [comment]
Hi @Asmedeus998 

For the formatting part, I also added some instructions here:  
https://github.com/ssricardo/anki-plugins/wiki/Fill-the-blank---Styles

For the input `std::cin`. I was trying to solve, but actually this will remain as a limitation of the add-on.
The problem is:  
Anki uses these *::*  to separate parts of the field declaration (type > cloze > field), so it's a reserved sequence. 
I was going to try to match the last *::*, then your value within the field wouldn't be considered complete. BUT then the problem is that it's would include every content until the closing of the last Cloze (that is, the last *::*). 
Summing up, it's would mess completely the add-on.

So, sorry but fixing is not possible.
